libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -41,14 +44,15 @@ typedef struct OVModel{
     ie_blob_t *input_blob;
 } OVModel;
 
-static const AVClass dnn_openvino_class = {
-    .class_name = "dnn_openvino",
-    .item_name  = av_default_item_name,
-    .option     = NULL,
-    .version    = LIBAVUTIL_VERSION_INT,
-    .category   = AV_CLASS_CATEGORY_FILTER,
+#define OFFSET(x) offsetof(OVContext, x)
+#define FLAGS AV_OPT_FLAG_FILTERING_PARAM | AV_OPT_FLAG_VIDEO_PARAM
```

Comment:
do we need AV_OPT_FLAG_VIDEO_PARAM?

Reply:
Sorry, I didn't take this seriously just copy it from vf_dnn_processing.c. Will check it later.